R/xrf.R

```diff
@@ -352,7 +354,8 @@ xrf.formula <- function(object, data, family,
 predict.xrf <- function(object, newdata,
                         sparse = TRUE,
                         lambda = 'lambda.min',
-                        type = 'response') {
+                        type = 'response',
+                        features = 'glm') {
```

Comment:
I'm not sure this belongs in the `predict` method - not the same functionality. I think it would make more sense to either:

-export the `evaluate_rules` function
-export a new function, named like `generate_xrf_design_matrix`, that is also used by `predict.xrf`

I think I prefer the second option.